src/main/java/com/hypherionmc/modpublisher/tasks/GithubUploadTask.java

```diff
@@ -127,13 +131,18 @@ public void upload() throws Exception {
 
             releaseBuilder.name(name);
 
+            // New releases should begin as drafts while we upload stuff to them
+            wasDraft = true;
+            releaseBuilder.draft(wasDraft);
+
             releaseBuilder.body(CommonUtil.resolveString(extension.getChangelog().get()));
-            releaseBuilder.draft(true);
-            releaseBuilder.commitish(ghRepository.getDefaultBranch());
+            releaseBuilder.commitish(Optional.ofNullable(extension.getGithub().getTarget()).orElseGet(ghRepository::getDefaultBranch));
```

Comment:
I believe this can be further simplified, because [GitHub's API](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) uses the default branch _by default_ when `target_commitish` is omitted.

```suggestion
            Optional.ofNullable(extension.getGithub().getTarget()).ifPresent(releaseBuilder::commitish);
```